README.md

```diff
@@ -643,9 +643,9 @@ For guidance on how to collect and use real-user data to debug performance issue
 
 _**⚠️ Warning ⚠️** the "base+polyfill" build is deprecated. See [#238](https://github.com/GoogleChrome/web-vitals/issues/238) for details._
```

Comment:
It seems kinda weird to be adding things to the polyfill, while also deprecating it in the next release.

But guess it's still "supported" for now but deprecated.

Reply:
I agree it's a bit weird.

In this case I was thinking of it as "deprecating this feature", and an easy way to do that was to move it to the polyfill build. But yeah, from a certain perspective it is making changes to a deprecated feature.

The good thing in this case is this is a polyfill, so it shouldn't cause any breakage for anyone's site. They'll just stop getting TTFB reports for IE and Safari 8-14.